config/locales/en.yml

```
@@ -31,3 +31,6 @@
 
 en:
   hello: "Hello world"
+  settings:
+    scores_locked: 'Scores have been locked.'
+    scores_unlocked: 'Scores have been unlocked.'
```

```suggestion
    scores_locked: 'Scores have been locked'
    scores_unlocked: 'Scores have been unlocked'
```
required^                loosely optional v
```suggestion
    scores_locked: 'Successfully locked scores'
    scores_unlocked: 'Successfully unlocked scores'
```